src/cmd/completion.go

```diff
@@ -43,13 +42,15 @@ func completionEmpty(_ *cobra.Command, _ []string, _ string) ([]string, cobra.Sh
 }
 
 func completionCommands(cmd *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
-	commandNames := []string{}
+	// init-container is hidden but documented command that is meant to be
+	// accessed by users at least via a manual page
```

Comment:
This comment is a bit confusing because one wonders what *hidden but documented* really is?  If it's hidden then why is it documented, and if it's documented then why is it hidden?

The main difference between `completion` and `init-container` is that the former is purely an implementation detail of our build that's of no interest to the user, while the latter does show up in things like `podman inspect`.

A user who is trying to troubleshoot or poke around at their Toolbx containers can be left wondering about `toolbox init-container ...`, which is a crucial detail of how Toolbx containers work.  Plus, the command does have some expectation of stability.  Otherwise existing Toolbx containers would break.  This is why it's worth documenting it.

However, `toolbox completion` is different.  It doesn't seem to show up in the generated shell completion files, and even if it did, it's not that fundamental to how Toolbx works.  Plus, there's really no need or expectation of stability.  It can change if Cobra changes.  Therefore, there's not much value in having a manual for it.

https://github.com/containers/podman/issues/14110 has a similar discussion but in the context of Podman, which does have a `podman-completion(1)` manual.